.github/ISSUE_TEMPLATE/bug_report.yaml

```
@@ -0,0 +1,50 @@
+name: "Bug Report (Low Priority)"
+description: "Create a public Bug Report. Note that these may not be addressed as quickly as the helpdesk and that looking up account information will be difficult."
+title: "[BUG]: "
+labels: type:bug
+body:
+  - type: input
+    attributes:
+      label: Tracer Version(s)
+      description: "Version(s) of the tracer affected by this bug"
+      placeholder: 1.9.0
+    validations:
+      required: true
+
+  - type: input
+    attributes:
+      label: Node.js Version(s)
+      description: "Version(s) of Node.js (`node --version`) that you've encountered this bug with"
+      placeholder: 20.1.1
+    validations:
+      required: true
+
+  - type: input
+    attributes:
+      label: Instrumentation Mode
+      description: "How is your application being instrumented"
+      placeholder: automatic with msi installer, manual with NuGet package
+    validations:
+      required: true
+
+  - type: input
+    attributes:
+      label: CLR
+      description: "What CLR and version are you using"
+      placeholder: ".NET Framework 4.6.2, .NET Core 2.1"
```

```suggestion
      label: TFM
      description: "What Target Framework Moniker (TFM) are you using in your project?"
      placeholder: "net462, net9.0"
```
I see that CLR was what we previously had, but I think this would be a bit more helpful and should be familiar for .NET developers.
I don't think CLR and version is correct and what we want to know, it's better to just know what TFM their project is targeting

For .NET Framework it'd be good to know what runtime version is installed as well but seems difficult to ask succinctly

---

Is there a finite set of answers for this? If so we can make it a dropdown.

---

Yes but there are many: https://learn.microsoft.com/en-us/dotnet/standard/frameworks#supported-target-frameworks

---

50 could be a little heavy...

---

> What Target Framework Moniker (TFM) are you using in your project

Ironically this is specifically _not_ what we want, we want the version they're running on 💀 We could change the wording around this, but tbh, they're probably going to give us the wrong answer on .NET FX regardless 🙈